### ISSUE TITLE
Installer doesn't work if CMS is not the main website page

### ISSUE THREAD

**Issue by BenjaminFarris** [Xtraball/Siberian#752]
I'm trying to install Siberian CMS on a seperate page of my website. Yet when I start the install all of the paths are off because they look like they are absolute paths and it looks like this. 
![image](https://user-images.githubusercontent.com/3847530/73878777-6fc7c700-4818-11ea-98a6-3d8a81497e63.png)

All of the elements are there but the paths are all absolute causing the entire thing to not function.

**Comment by SiberianCMS** [comment]
Siberian must be installed on the web root, it can't be inside a subfolder.

**Comment by BenjaminFarris** [comment]
It looks like it should be pretty easy to fix it, so I'm going to take a crack at it. Any recommendations on where to look to change the path?

**Comment by Neustradamus** [comment]
@BenjaminFarris: One month after, have you looked?

**Comment by BenjaminFarris** [comment]
@Neustradamus I gave it a try for a bit. I could get the installer page to download everything correctly but once I tried running the installer it looked like the path to the php library wasn't pointed to correctly. I didn't know what I was looking for so I eventually gave up. I think it's possible but someone with a deeper understanding of the codebase or more time is needed. At this point I tried installing on a fresh server I spun up for it, and things still didn't work so i moved on to other things
